api/envoy/service/accesslog/v3/als.proto

```diff
@@ -84,4 +105,8 @@ message StreamAccessLogsMessage {
 
     TCPAccessLogEntries tcp_logs = 3;
   }
+
+  // This is an ID to identify the message, and can be added to the Critical Endpoint
+  // response message to uniquely identify the message being buffered in the Envoy.
+  uint32 id = 4;
```

Comment:
Would it be nicer to not include this in the existing message? Could we for example have a BufferedAccessLogsMessage that wraps a StreamAccessLogsMessage and adds the id?

Reply:
That makes sense. Fixed.